packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -37,6 +37,8 @@ import { AmountInput } from '../util/AmountInput';
 // import { DragDrop, Draggable, Droppable, DragDropHighlight } from './dragdrop';
 import { ListItem, ROW_HEIGHT } from './MobileTable';
 
+const smallVPWidth = document.documentElement.clientWidth < 365;
```

Comment:
You can use `useResizeObserver` to listen for screen size changes and then show the blocks depending on that size.

Sample: https://github.com/actualbudget/actual/blob/21d5f117eec2e881c0dbc03ff85f02d63fccf3a8/packages/desktop-client/src/components/budget/MonthPicker.js#L43

Reply:
I think this is my last comment. Once we have this solved I'll update the screenshots and I think we're good to go :)

Reply:
Unfortunately that will not work as its required to be in a function and not a class. As of right now, there is support for iphone SE and I truly don't believe people are going to resize their desktop screen less than 370px. If anyone wants to take over and add dynamic support, they can.